site/docs/reference/Connectors/capture-connectors/sqlserver.md

```diff
@@ -0,0 +1,277 @@
+Setup instructions are provided for the following platforms:
+
+* [Self-hosted SQL Server](#setup-self-hosted-sql-server)
+* [Azure SQL Database](#setup-azure-sql-database)
+* [Amazon RDS for SQL Server](#setup-amazon-rds-for-sql-server)
+* [Google Cloud SQL for SQL Server](#setup-google-cloud-sql-for-sql-server)
+
+## Prerequisites
+
+To capture change events from SQL Server tables using this connector, you need:
+
+* For each table to be captured, a primary key should be specified in the database.
+If a table doesn't have a primary key, you must manually specify a key in the associated Flow collection definition while creating the capture.
+[See detailed steps](#specifying-flow-collection-keys).
+
+* [CDC enabled](https://learn.microsoft.com/en-us/sql/relational-databases/track-changes/enable-and-disable-change-data-capture-sql-server?view=sql-server-ver16)
+on the database and the individual tables to be captured.
+(This creates *change tables* in the database, from which the connector reads.)
+
+* A **watermarks table**. The watermarks table is a small “scratch space” to which the connector occasionally writes a small amount of data to ensure accuracy when backfilling preexisting table contents.
+
+* A user role with:
+  * `SELECT` permissions on the CDC schema and the schemas that contain tables to be captured.
+  * `SELECT` and `UPDATE` permissions on the watermarks table
+
+To meet these requirements, follow the steps for the your hosting type.
+
+* [Self-hosted SQL Server](#setup-self-hosted-sql-server)
+* [Azure SQL Database](#setup-azure-sql-database)
+* [Amazon RDS for SQL Server](#setup-amazon-rds-for-sql-server)
+* [Google Cloud SQL for SQL Server](#setup-google-cloud-sql-for-sql-server)
+
+### Setup: Self-hosted SQL Server
+
+1. Connect to the server and issue the following commands:
+
+```sql
+USE <database>;
+--Enable CDC for the database.
+EXEC sys.sp_cdc_enable_db;
+--Create user and password for use with the connector.
+CREATE LOGIN flow_capture WITH PASSWORD = 'secret';
+CREATE USER flow_capture FOR LOGIN flow_capture;
+/*Grant the user permissions on the CDC schema and schemas with data.
+This assumes all tables to be captured are in the default schema, `dbo`.
+Add similar queries for any other schemas that contain tables you want to capture.*/
+GRANT SELECT ON SCHEMA :: dbo TO flow_capture;
+GRANT SELECT ON SCHEMA :: cdc TO flow_capture;
+--Create the watermarks table, grant permissions, and enable CDC.
+CREATE TABLE dbo.flow_watermarks(slot INTEGER PRIMARY KEY, watermark TEXT);
+GRANT SELECT, UPDATE ON dbo.flow_watermarks TO flow_capture;
+EXEC sys.sp_cdc_enable_table @source_schema = 'dbo', @source_name = 'flow_watermarks', @role_name = 'flow_capture', @capture_instance = 'dbo_flow_watermarks';
+--Add some data to watermarks table to ensure proper connector initiation.
+INSERT INTO dbo.flow_watermarks VALUES (0, 'dummy-value');
+```
+
+2. Set up an [SSH server for tunneling](../../../../guides/connect-network/).
```

Comment:
Should we have an "Alternatively you can whitelist our IP address in your firewall rules" note here, for symmetry with how the managed DB setup instructions call out the SSH tunneling option?

Reply:
Thanks - for some reason I'd been under the impression this was less of a viable option here than on managed cloud platforms but now I think about it I don't see why it would be...